From 6f034c974efc3a48bae27cce6bfc6ef697e2cfb5 Mon Sep 17 00:00:00 2001
From: zhangyichix <yichix.zhang@intel.com>
Date: Thu, 3 Aug 2023 14:07:06 +0800
Subject: [PATCH] Disabled Omx

android.hardware.media.omx@1.0::IOmxStore/default is deprecated
in compatibility matrix at FCM Version 8, so we disabled it.

Tracked-On: OAM-110599
Signed-off-by: zhangyichix <yichix.zhang@intel.com>
---
 groups/device-specific/caas/manifest.xml | 15 +--------------
 1 file changed, 13 deletions(-)

diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index 6a2f4eb..0bb5226 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -63,19 +63,6 @@
             <instance>default</instance>
         </interface>
     </hal-->
-    <hal format="hidl">
-        <name>android.hardware.media.omx</name>
-        <transport>hwbinder</transport>
-        <version>1.0</version>
-        <interface>
-            <name>IOmx</name>
-            <instance>default</instance>
-        </interface>
-        <interface>
-            <name>IOmxStore</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
     <!--hal format="hidl">
         <name>android.hardware.drm</name>
         <transport>hwbinder</transport>
-- 
2.40.0

